File: R/tests/testthat/test_banditNarm_4par.R
context("Test banditNarm_4par")
library(hBayesDM)

test_that("Test banditNarm_4par", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm_4par(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_banditNarm_4par.py
import pytest

from hbayesdm.models import banditNarm_4par


def test_banditNarm_4par():
    _ = banditNarm_4par(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
